src/main/scala/org/allenai/wikitables/WikiTablesSemanticParserCli.scala

```diff
@@ -25,105 +12,111 @@ import com.jayantkrish.jklol.ccg.lambda2.SimplificationComparator
 import com.jayantkrish.jklol.cli.AbstractCli
 import com.jayantkrish.jklol.nlpannotation.AnnotatedSentence
 import com.jayantkrish.jklol.training.DefaultLogFunction
-import com.jayantkrish.jklol.util.CountAccumulator
-import com.jayantkrish.jklol.util.IndexedList
 import edu.cmu.dynet._
+import edu.stanford.nlp.sempre.LanguageAnalyzer
+import edu.stanford.nlp.sempre.tables.TableKnowledgeGraph
 import edu.stanford.nlp.sempre.tables.test.CustomExample
 import joptsimple.OptionParser
 import joptsimple.OptionSet
 import joptsimple.OptionSpec
+
+import org.allenai.pnp.Env
+import org.allenai.pnp.LoglikelihoodTrainer
+import org.allenai.pnp.PnpExample
+import org.allenai.pnp.PnpModel
 import org.allenai.pnp.semparse.ActionSpace
+import org.allenai.pnp.semparse.ConstantTemplate
+import org.allenai.pnp.semparse.Entity
+import org.allenai.pnp.semparse.EntityLinking
+import org.allenai.pnp.semparse.SemanticParser
 import org.allenai.pnp.semparse.SemanticParserConfig
+import org.allenai.pnp.semparse.SemanticParserUtils
 
-/** Command line program for training a semantic parser 
+/** Command line program for training a semantic parser
   * on the WikiTables data set.
   * runMain org.allenai.wikitables.WikiTablesSemanticParserCli -trainingData TRAIN-DATA-PATH
   *                                                           [-testData TEST-DATA-PATH]
   *                                                           [-derivationsPath PATH-TO-LOGICAL-FORMS]
   * If derivationsPath is not specified, Sempre will be used to parse utterances (this will be SLOW!)
   */
 class WikiTablesSemanticParserCli extends AbstractCli() {
-  
-  import WikiTablesSemanticParserCli._
-  
+
+  import WikiTablesUtil._
+
   var trainingDataOpt: OptionSpec[String] = null
   // Path to the directory containing the correct logical forms
   var derivationsPathOpt: OptionSpec[String] = null
   var modelOutputOpt: OptionSpec[String] = null
-  
+
   var maxDerivationsOpt: OptionSpec[Integer] = null
   var epochsOpt: OptionSpec[Integer] = null
   var beamSizeOpt: OptionSpec[Integer] = null
-  
+
   var skipActionSpaceValidationOpt: OptionSpec[Void] = null
   var trainOnAnnotatedLfsOpt: OptionSpec[Void] = null
 
+  // Initialize expression processing for Wikitables logical forms.
+  val simplifier = ExpressionSimplifier.lambdaCalculus()
+  val comparator = new SimplificationComparator(simplifier)
+  val logicalFormParser = ExpressionParser.expression2();
+  val typeDeclaration = new WikiTablesTypeDeclaration()
+
+  CustomExample.opts.allowNoAnnotation = true;
+  TableKnowledgeGraph.opts.baseCSVDir = "data/WikiTableQuestions";
+  LanguageAnalyzer.opts.languageAnalyzer = "corenlp.CoreNLPAnalyzer";
```

Comment:
Is this the right place to set these flags? E.g., will we have to do this in every CLI we write? Will we ever want different choices for these? The other option is to set them in WikiTablesUtil or WikiTablesDataProcessor. I don't know what the right thing to do is, so I'll leave the decision to you.